CMakeLists.txt

```diff
@@ -262,8 +262,8 @@ if(BUILD_DEV)
 else()
     set(MIOPEN_BUILD_DEV 0)
     set(MIOPEN_SYSTEM_DB_PATH "${CMAKE_INSTALL_PREFIX}/${DATA_INSTALL_DIR}/db" CACHE PATH "Default path of system db files")
-    set(MIOPEN_USER_DB_PATH "~/.config/miopen/" CACHE STRING "Default path of user db files")
-    set(MIOPEN_CACHE_DIR "~/.cache/miopen/" CACHE STRING "")
+    set(MIOPEN_USER_DB_PATH "~/.config/miopen.${MIOpen_VERSION_MAJOR}_${MIOpen_VERSION_MINOR}_${MIOpen_VERSION_PATCH}_${MIOpen_VERSION_TWEAK}/" CACHE STRING "Default path of user db files")
```

Comment:
I dont think we should version the db here. We should only version the finddb(since its fast to compute). We shouldn't version the perfdb. No one wants to spend a week to re-tune because only the rocm version changed. 

Users of multi-rocm will most likely not use perfdb(and we make it clear not use it in reproducible multi-install environments) since it requires extra special steps to create. The finddb, it makes sense to version as that will get updated by when users call `Find`.